src/FixedDataTableStore.js

```diff
@@ -12,9 +12,15 @@
 'use strict';
 
 import reducers from 'reducers';
-import { createStore } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
 
-const store = createStore(reducers);
+const store = configureStore({
+  reducer: reducers,
+  middleware: getDefaultMiddleware => getDefaultMiddleware({
+    serializableCheck: false,
+    immutableCheck: false
```

Comment:
State is a circular data structure, hence immutable check will always result in an infinite loop. Have to disable this as well.

Reply:
why is this circular?